### PR TITLE
revert: bulk submit

### PR DESCRIPTION
**Pull request by nsprenkle**
**TL;DR -** we didn't ever end up using bulk grade submit and it just made things more complicated. Remove it.

JIRA: [AU-558](https://openedx.atlassian.net/browse/AU-558)

**What changed?**

- This (loosely) reverts commit https://github.com/openedx/edx-ora2/commit/70bfc6e0a7bb20046c0ec14de131ab4114fc7255
- Keep some refactors on top of those changes.
- Build/translations stuffs.

**Developer Checklist**

- [x] Reviewed the [release process](https://github.com/edx/edx-ora2/blob/master/.github/release_process.md)
- [x] Translations and JS/SASS compiled
- [x] Bumped version number in [setup.py](https://github.com/edx/edx-ora2/blob/a62e81a9b0d89223476967ec3c27f3557a850735/setup.py#L39) and [package.json](https://github.com/edx/edx-ora2/blob/a62e81a9b0d89223476967ec3c27f3557a850735/package.json#L3)

**Testing Instructions**

- [ ] Unit tests pass
- [ ] Smoke tests pass

**Reviewer Checklist**

Collectively, these should be completed by reviewers of this PR:

- [ ] I've done a visual code review
- [ ] I've tested the new functionality

FYI: @openedx/content-aurora

### CODE REVIEW

**Comment by jansenk** [comment]
:(

**Comment by nsprenkle** [comment]
> :(

I know sad, but if it means we can remove 500 lines of code I'm calling it a win.

**Comment by nsprenkle** [comment]
Can safely ignore codecov because it's complaining about exception cases not being covered but they were never covered.

**Review by leangseu-edx — APPROVED** [review]
LGTM

**Review by mattcarter — APPROVED** [review]
approved
